### ISSUE TITLE
not seeing all of my tasks

### ISSUE THREAD

**Issue by hymie0** [coddingtonbear/inthe.am#244]
I am not seeing all of my tasks in Inthe.AM web site.

Specifically, tasks that have two tags ("remind" "sdo") are not appearing.

I couldn't find anything in the config settings that might relate to multiple tags, so I don't know if I did something wrong.

**Comment by coddingtonbear** [comment]
I have no idea what might be preventing your tasks from appearing, @hymie0, but I did have a series of production problems over the last couple of weeks.  Could you try clearing your task list on Inthe.AM (following these instructions https://github.com/coddingtonbear/inthe.am/issues/250) and giving it another shot?

**Comment by hymie0** [comment]
That appears to have resolved the problem.  Thanks.

**Comment by hymie0** [comment]
This is happening again.  I did the resync and all of my current tasks were loaded.  But I had some tasks that were "waiting", and either they weren't sync'd, or they aren't being un-waited when the time comes.  I'm not sure how to tell which.

**Comment by coddingtonbear** [comment]
Hrm; I can't think of a reason for such a thing to be happening, @hymie0.  Could you let me know of the ID# of a task you're expecting to see but are not; if you do, I can look at your task list on the server to try to figure out what the hangup might be.

**Comment by hymie0** [comment]
```
Name          Value                                                           
ID            7
Description   install the backup disk                                         
Status        Pending
Entered       2016-08-31 16:02:12 (7w)                                        
Scheduled     2016-10-25 00:00:00
Due           2016-10-25 09:00:00                                             
Last modified 2016-08-31 16:02:12 (7w)
Tags          sdo remind                                                      
Virtual tags  DUE MONTH PENDING SCHEDULED TAGGED TOMORROW UNBLOCKED WEEK YEAR
UUID          da5e94e5-127f-4972-bd10-0d2d4ee71ca5                            
```

**Comment by coddingtonbear** [comment]
I think you might be right, and the only reason I hadn't noticed it is that I mostly operate on my task list locally.  The process via which tasks are moved from the `waiting` to `pending` state (and from your `pending.data` to `closed.data` among other things) is called "garbage collection", and I've disabled it in most cases on Inthe.AM to prevent task listings from needing to acquire an exclusive lock on your task list.

Garbage collection, though, is still necessary sometimes, and I've configured Inthe.AM to attempt to trick Taskwarrior into running its garbage collection routine whenever any task operation takes place that might alter your task list.  That functionality, though, appears to have been broken!

I'm working on a fix, though, and I'll keep you posted!
